.travis.yml

```diff
@@ -35,10 +36,9 @@ jobs:
         cp -r documentation/docs documentation/docs-tmp
         documentation/bin/createDocu.sh vars documentation/docs-tmp/steps
         docker run --rm -it -v ${TRAVIS_BUILD_DIR}:/docs -w /docs/documentation squidfunk/mkdocs-material:3.0.4 build --clean --verbose --strict
-    - name: Consumer Tests for s4sdk pipeline (CloudFoundry)
-      script: cd consumer-test/s4sdk/CloudFoundry && chmod +x runTests.sh && ./runTests.sh
-    - name: Consumer Tests for s4sdk pipeline (Neo Environment)
-      script: cd consumer-test/s4sdk/NeoEnvironment && chmod +x runTests.sh && ./runTests.sh
+    - name: Consumer Tests
```

Comment:
Up to now the approach was to add each new test here.

This has two disadvantages:
1.) each time we provide a new test we have to adjust our file controlling the build
2.) with that approach it is hard to synchronized over all the tests. But we need that in the future since we agreed to sending a status notification when all build are done.

Disadvantage with the new approach:
Before it was possible to drill down in the individual tests and to review the log while it was written. Now - since we do not have particular stages for each test - this is not possible anymore. The build logs are `cat`' ed in the big log when all the tests are finished. IMO this is only a minor disadvantage since normally nobody has a direct look on the integration test while they are running.